src/realm/sync/network/default_socket.cpp

```diff
@@ -94,6 +94,7 @@ class DefaultWebSocketImpl final : public DefaultWebSocket, public Config {
     {
         WebSocketError error = WebSocketError::websocket_ok;
         bool was_clean = true;
+        std::string msg = body.empty() ? ec.message() : std::string{body};
```

Comment:
This could avoid copying the body if it was inline in the call to websocket_error_and_close_handler() as the temporary from message() would live long enough, but I guess that doesn't actually matter.